autoware_launch/config/system/default_adapi/default_adapi.param.yaml

```
@@ -0,0 +1,45 @@
+/**:
+  ros__parameters:
+    diagnostic_updater:
+      period: 1.0
+      use_fqn: true
+
+/adapi/node/autoware_state:
+  ros__parameters:
+    update_rate: 10.0
+
+/adapi/node/motion:
+  ros__parameters:
+    require_accept_start: false
+    stop_check_duration: 1.0
+
+/adapi/node/routing:
+  ros__parameters:
+    stop_check_duration: 1.0
+
+/adapi/node/vehicle_metrics:
+  ros__parameters:
+    update_rate: 0.1
+    max_energy_level: 100.0
```

Looking [this `autoware_universe` side](https://github.com/tier4/autoware_universe/blob/ca280f42e26be75262282d113f2f7a75424a7b86/system/autoware_default_adapi_universe/config/default_adapi.param.yaml#L23), I guess
```suggestion
    max_energy_level: 1.0
```

---

Yes, Pilot.Auto uses percentage so API node converts it to ratio.